### PR TITLE
chore: change semantic-release command in github action steps

### PR DESCRIPTION
**Pull request by charan678**
# Fixes #

Semantic release step failed during our last release process and failure happened when It was trying to reading current tag version and returned NOTAGNAME and INVALIDTAGNAME error codes. 
In this fix I changed the command from npx to npm bin command which worked. I don't the reason why later one worked as compare to former one.  I have rasied the issue on semantic-release repo (https://github.com/semantic-release/semantic-release/issues/2297) to know more about it.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -62,14 +62,14 @@ jobs:
       - name: Semantic Release runs for draft release
         if: ${{github.event.inputs.draft-release == 'true'}}
         id: semantic-release-draft
-        run: npx semantic-release -t \${version}
+        run: DEBUG=semantic-release:* $(npm bin)/semantic-release  -t \${version}
```
Comment (onuzbee) [inline .github/workflows/release.yml:65]: Can you add more context why this changes are needed and how it solves the issue?

**Comment by shrutiburman** [comment]
Can you link the issue in "Fixes" heading and workflow run links done for testing?

File: .github/workflows/release.yml (hunk 1 above)
Comment (charan678) [reply .github/workflows/release.yml:65]: I don't know why doesn't work with npx command. This is the issue https://github.com/semantic-release/semantic-release/issues/2297 which I raised on semantic-release repo.

Comment (onuzbee) [reply .github/workflows/release.yml:65]: Okay, So we're just adding debug logs for it? 

Also - Should it be `${npm bin}` instead of `$(npm bin)`?

Comment (charan678) [reply .github/workflows/release.yml:65]: https://stackoverflow.com/questions/27472540/difference-between-and-in-bash/27472808
It should $(npm bin) and yes we added debug for better logs

**Review by shamantraghav — APPROVED** [review]

**Review by onuzbee — COMMENTED** [review]
:+1
